cppcache/CMakeLists.txt

```diff
@@ -41,6 +41,14 @@ set_target_properties(version-header PROPERTIES
 	FOLDER cpp
 )
 
+if (WIN32)
+  add_compile_definitions(WIN32)
+endif()
+
+IF(MSVC)
+    SET(CMAKE_CXX_FLAGS "/EHsc")
```

Comment:
This should be set at the root project so all inherit now if it’s required on all. Otherwise I think there is a set target property for this. At the very least if you’re changing this variablenit should be appended to the current value and not replacing.

Reply:
Moved this to root.